scripts/RetentionScript/README.md

```diff
@@ -0,0 +1,68 @@
+
+# About the scripts:
+These scripts are used to cleanup the storage configured for BRS service in service fabric. It allows you to delete data older than specific date time.
```

Comment:
Reword: Service Fabric Backup Restore Service (share link), currently in preview, allows you to take periodic backups of your stateful service. Depending the on the frequency of your backup interval, backups can really grow fast. While we work on providing retention support integrated with the service, this script would help you manage your storage till that time. It allows you to delete backups older than a specified time.